src/test/java/ezvcard/io/json/JCardSerializerTest.java

```diff
@@ -0,0 +1,159 @@
+package ezvcard.io.json;
+
+import static ezvcard.util.StringUtils.NEWLINE;
+import static org.junit.Assert.assertEquals;
+
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import ezvcard.VCard;
+
+/**
+ * @author Buddy Gorven
+ */
+public class JCardSerializerTest {
+
+	@Test
+	public void serialize_indented() throws Throwable {
+		VCard vcard1 = new VCard();
+		vcard1.setFormattedName("John Doe");
+
+		VCard vcard2 = new VCard();
+		vcard2.setFormattedName("John Doe");
+
+		ObjectMapper mapper = getMapper();
+
+		final DefaultPrettyPrinter pp = new DefaultPrettyPrinter();
```

Comment:
I didn't know about this pretty-printer class.  You probably noticed that in `JCardRawWriter` I tried to do my own pretty printing.  It doesn't work too well.  Do you think you could integrate jackson's pretty-printing system into `JCardRawWriter`?  Or is this something that can only be done at the "ObjectMapper" level?